shell/platform/android/test/io/flutter/embedding/android/FlutterViewTest.java

```diff
@@ -92,6 +92,15 @@ public void attachToFlutterEngine_alertsPlatformViews() {
     verify(platformViewsController, times(1)).attachToView(flutterView);
   }
 
+  @Test
+  public void flutterView_importantForAutofillDoesNotExcludeDescendants() {
```

Comment:
Is it possible to test not that the value is set but instead that a child view can have autofill? 
This feels a bit like a change detector test.

Reply:
Does this test need to indicate that it should only run for android o and above?

Reply:
> Is it possible to test not that the value is set but instead that a child view can have autofill?
This feels a bit like a change detector test.

I wasn't able to find a way to test if child views receive autofills automatically. I tried to manually call [autofill](https://developer.android.com/reference/android/view/View#autofill(android.view.autofill.AutofillValue)) with no luck. And the documentation only provides an example to test it manually: https://developer.android.com/guide/topics/text/autofill-optimize.

> Does this test need to indicate that it should only run for android o and above?

I believe the method is only callable on versions O and above. Without the version check, we would get lint warnings that the method would cause crashes on lower versions.